src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  --link_color: #00acdc;
+}
+
+.header_moyo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+img {
+  display: block;
+}
+
+.nav_moyo {
+  height: 60px;
+}
+
+.nav_moyo ul {
```

Comment:
Consider using classes as selectors

Reply:
I don't understand what I have to change in these lines? I use classes as selectors. Maybe you mean that I should use a class for <ul>?

Reply:
for tag ul?